src/esp_idf_svc.rs

```diff
@@ -0,0 +1,2 @@
+#[cfg(feature = "esp32")]
```

Comment:
Here is where we re-export `esp-idf-svc`. Note that the `feature = "esp32"` condition is duplicated between this file and the `pub mod` statement in `src/lib.rs`. I wasn't sure whether it was better to make the `pub mod` conditional, or this conditional, or both. I opted for both, but please let me know if you have a preference for removing one or the other.

Reply:
if you move the file to esp32 does that help? If so then we can keep them  under crate::esp32 namespace

Reply:
Sure, I can move it down a layer, though it will result in a somewhat unwieldy name like `crate::esp32::esp_idf_svc::[sys|hal]::`, or `micro_rdk::esp32::esp_idf_svc::[sys|hal]::`. If we wanted to remove the redundant `esp32::esp` I could probably re-export it as `idf_svc`? So it'd be `[crate|micro_rdk]::esp32::idf_svc::[sys|hal]::`. On the other hand, I feel like renaming the crate is iffy. Happy to do it any way, just let me know.